modules/tensor_mechanics/src/materials/ComputeCappedWeakPlaneStress.C

```diff
@@ -389,7 +389,7 @@ ComputeCappedWeakPlaneStress::dVardTrial(bool elastic_only, Real q, Real q_trial
   const int gesv_num_pq = _num_pq;
   LAPACKgesv_(&gesv_num_rhs, &gesv_num_pq, &jac[0], &gesv_num_rhs, &ipiv[0], &rhs_cto[0], &gesv_num_rhs, &info);
   if (info != 0)
-    throw MooseException("ComputeCappedWeakPlaneStress: PETSC LAPACK gsev routine returned with error code " + Moose::stringify(info));
+    errorHandler("ComputeCappedWeakPlaneStress: PETSC LAPACK gsev routine returned with error code " + Moose::stringify(info));
```

Comment:
Yep, you're right - if i left the "const" things "const" then i wouldn't need this errorHandler.  But for the moment i feel the const_cast is the best solution.